sktime/forecasting/ets.py

```diff
@@ -416,63 +416,26 @@ def _predict(self, fh, X=None, **simulate_kwargs):
         y_pred.name = self._y.name
         return y_pred.loc[valid_indices]
 
-    def _predict_interval(self, fh, X=None, coverage=None):
-        """Compute/return prediction quantiles for a forecast.
-
-        private _predict_interval containing the core logic,
-            called from predict_interval and possibly predict_quantiles
-
-        State required:
-            Requires state to be "fitted".
-
-        Accesses in self:
-            Fitted model attributes ending in "_"
-            self.cutoff
+    @staticmethod
+    def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
+        """Construct confidence interval at specified `alpha` for each timestep.
 
         Parameters
         ----------
-        fh : guaranteed to be ForecastingHorizon
-            The forecasting horizon with the steps ahead to to predict.
-        X : optional (default=None)
-            guaranteed to be of a type in self.get_tag("X_inner_mtype")
-            Exogeneous time series to predict from.
-        coverage : list of float (guaranteed not None and floats in [0,1] interval)
-           nominal coverage(s) of predictive interval(s)
+        prediction_results : PredictionResults
+            results class for predictions
+        alpha : float
+            level of significance
```

Comment:
for confidence intervals, should this be "nominal coverage"? Or is it "one minus nominal coverage"?
"level of significance" I think is not accurate, as we are not conducting a frequentist hypothesis test here.

Reply:
This is the [documentation](https://www.statsmodels.org/dev/generated/statsmodels.tsa.base.prediction.PredictionResults.conf_int.html#statsmodels.tsa.base.prediction.PredictionResults.conf_int) for the underlying `statsmodels` class, didn't want to change:

> alpha[float](https://docs.python.org/3/library/functions.html#float), optional
The significance level for the prediction interval. The default alpha = .05 returns a 95% confidence interval.

Reply:
I really wonder what proportion of users will know/care about the distinction between the two. Admittedly, visualising the difference between ranges of future given past and ranges of expected values of future given past is difficult for me (even assuming I am doing it correctly.)

But I don't think that is a reason to affect the docstring, especially because it's a private method. Updated with your suggestion.